customcommands/tmplextensions.go

```diff
@@ -180,11 +180,20 @@ func tmplRunCC(ctx *templates.Context) interface{} {
 			return "", templates.ErrTooManyCalls
 		}
 
-		cmd, err := models.FindCustomCommandG(context.Background(), ctx.GS.ID, int64(ccID))
+		// cmd, err := models.FindCustomCommandG(context.Background(), ctx.GS.ID, int64(ccID))
+		// if err != nil {
+		// 	return "", errors.New("Couldn't find custom command")
+		// }
+
```

Comment:
You can remove this commented-out code completely, and likewise below. Otherwise everything looks OK to me.

Reply:
yup :)